app/uk/gov/hmrc/economiccrimelevyregistration/controllers/CheckYourAnswersController.scala

```diff
@@ -107,29 +105,43 @@ class CheckYourAnswersController @Inject() (
           eclDetails(),
           organisationDetails(),
           contactDetails(),
+          otherEntityDetails(),
+          !request.registration.otherEntityJourneyData.isBlank(),
```

Comment:
It looks like we only care about the case where it's defined, to avoid the negation you could change the function to `isDefined` and check for` _.isDefined `instead of `_.isEmpty`?

Reply:
Done.